include/erlcloud_ecs.hrl

```diff
@@ -247,21 +259,51 @@
     task_role_arn :: undefined | binary() 
 }).
 
+-record(ecs_attachment_detail, {
+    name :: undefined | binary(),
+    value :: undefined | binary()
+}).
+
+-record(ecs_attachment, {
+    id :: undefined | binary(),
+    type :: undefined | binary(),
+    status :: undefined | binary(),
+    details :: undefined | [#ecs_attachment_detail{}]
+}).
+
+-record(ecs_tag, {
+    key :: undefined | binary(),
+    value :: undefined | binary()
+}).
+
 -record(ecs_task, {
+    attachments :: undefined | [#ecs_attachment{}],
+    availability_zone :: undefined | binary(),
     cluster_arn :: undefined | binary(),
+    connectivity :: undefined | binary(),
+    connectivity_at :: undefined | number(),
```

Comment:
This is a [Timestamp](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Task.html#ECS-Type-Task-connectivityAt), I assume that's actually... a `binary()` for us? Example says so.